### DISCUSSION TITLE
The command line for installation without any action, but why?!

### DISCUSSION THREAD

**Discussion by TorbenSchmitz** [mcguirepr89/BirdNET-Pi#1192] [General]
Hi there, 
I installed a few birdnetpis already. This time got stuck a the very beginning. I followed all installation steps (bullseye lite, ssh enabling, ...). I am connected via ethernet cable and tried via PuTTY and windows 11 console.
The installation command line remains without any action. I am happy receiving some help. 

`PS C:\Users\xx.xx> ssh Birder@birdnetpi.local
Birder@birdnetpi.local's password:
Linux birdnetpi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64

The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
Last login: Tue Mar 12 02:18:17 2024 from fe80::d48b:841e:5828:b461%eth0
Birder@birdnetpi:~ $ curl -s https://raw.githubusercontent.com/mcguirepr89/BirdNET-Pi/main/newinstaller.sh | bash
Birder@birdnetpi:~ $`

**Comment by gilpayson** [comment]
I get the a different error, but over and over and over: TFlite.......whl fails.  Apparently there is some 32-bit code stuck somewhere.

What is your hardware?  I am using Raspberry pi 5 with 8gb ram

**Reply by TorbenSchmitz** [reply]
I am using a Raspberry Pi 4B 4GB, as before. 

I wonder why the command line is answered with a new command prompt. No errors displayed. Just ignoring my input :)

**Reply by TorbenSchmitz** [reply]
Solved.

I was preparing the pi for a dedicated wlan network. During the preparation I set it up via lan network cable. obviously it was awaiting the communication via presetted wlan. 

Putting it in the wlan network and initializing the installation via command line, it worked perfectly.

**Reply by gilpayson** [reply]
I have 1)  dedicated lan with 1GB BW   2)  Brand new SD, brand new Raspberry 5, 8GB.  

PLS provide the exact URL you used for boot img.  I have tried many boot files, but cannot find an exactly corresponding to
Linux birdnetpi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr 3 17:24:16 BST 2023 aarch64.

You are using an old IMG.

Mine is:
Linux WhatzIt 6.6.28+rpt-rpi-2712 #1 SMP PREEMPT Debian 1:6.6.28-1+rpt1 (2024-04-22) aarch64 GNU/Linux

I would like to try out your img!  - Please try it with the latest images:  looks as though there is a broken update.
